ghost/staff-service/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@tryghost/staff-service",
+  "version": "0.0.0",
+  "repository": "https://github.com/TryGhost/Ghost/tree/main/packages/staff-service",
+  "author": "Ghost Foundation",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "dev": "echo \"Implement me!\"",
+    "test:unit": "NODE_ENV=testing c8 --all --check-coverage --reporter text --reporter cobertura mocha './test/**/*.test.js'",
+    "test": "yarn test:unit",
+    "lint:code": "eslint *.js lib/ --ext .js --cache",
+    "lint": "yarn lint:code && yarn lint:test",
+    "lint:test": "eslint -c test/.eslintrc.js test/ --ext .js --cache"
+  },
+  "files": [
+    "index.js",
+    "lib"
+  ],
+  "devDependencies": {
+    "mocha": "10.0.0",
+    "should": "13.2.3",
+    "sinon": "14.0.0"
+  },
+  "dependencies": {
+    "lodash": "4.17.21",
+    "moment": "2.29.1"
```

Comment:
Not sure about the dependencies in the monorepo, but `handlebars` might be missing here (though I think it should work without it).

Reply:
🤔 Yea this is a weird one, because I don't see `handlebars` as a direct dependency for any of the packages including `core`, though `comments` service also uses it. Not sure about adding it separately here when we haven't done so far, but will double check on this for whats the expected behavior.

Reply:
Pinned handlebars to `"handlebars": "4.7.7"` which seems the right version here.